### ISSUE TITLE
Json importado a postman

### ISSUE THREAD

**Issue by Regenschrim** [IIC2413/Syllabus-2018-2#151]
No tengo claro cual es la función del archivo "Ayudantia Flask.post_collection.json" que subieron en la ayudantía. Solo entendí que se subía a Postman para poder probar la API que se creo en el archivo "__init__.py". ¿También hay que crear una archivo de ese estilo para la tarea?
Gracias!

**Comment by cmp831** [comment]
No, Postman es un programa para que puedan hacer requests o testear su API con mayor facilidad. Para la entrega no es necesario entregar el archivo para Postman

**Comment by beayancan** [comment]
De todas formas para saberlo, ¿Qué es lo que hace el archivo en general?

**Comment by valerojasm** [comment]
Hola! Entonces hay que hacer una interfaz además de la api para probarla o la probarán con postman?

**Comment by cmp831** [comment]
@valerojasm Nono, como dije anteriormente, Postman solo facilita el testeo para ustedes, para la entrega no es necesario entregar nada de Postman, tampoco se pide una interfaz.

**Comment by cmp831** [comment]
@beayancan El archivo guarda la colección que creaste en postman, una colección es simplemente un conjunto de requests que puedes hacer en Postman. Además de guardar las rutas de los requests, también guarda los variables que se mandan, junto con la JSON si existe para una request particular. Al importar el archivo, pueden tener la misma colección que se hizo en la ayudantía, era para que entiendan mejor la aplicación que se creó, pero al parecer no lo expliqué muy bien jaja.
